src/org/stringtemplate/v4/misc/ObjectModelAdaptor.java

```diff
@@ -37,7 +37,7 @@
 import java.util.HashMap;
 import java.util.Map;
 
-public class ObjectModelAdaptor implements ModelAdaptor {
+public class ObjectModelAdaptor implements ModelAdaptor<Object> {
```

Comment:
Perhaps we should consider making ObjectModelAdaptor generic as well, inspired by this part in the docs:

> You can inherit your ModelAdaptor from ObjectModelAdaptor to leverage its ability to handle "normal" attributes. You can choose to override the results of any given property or to handle properties that would not normally be handled by the default ObjectModelAdaptor.

@sharwell, what do you think?

Reply:
That would address the breaking change too, right?

Reply:
You mean the one mentioned in the PR description? No, as that only applies to `MapModelAdaptor` and `AggregateModelAdaptor`.

Reply:
No objection to making ObjectModelAdaptor generic

Reply:
Done in dd043d24f6a9386e2c37e7fd8836861abe42dd21